projects/saturn/src/main/java/io/fairspace/saturn/config/Config.java

```diff
@@ -102,6 +102,7 @@ public static class Caches {
     public static class Search {
         public long pageRequestTimeout = 10_000;
         public long countRequestTimeout = 100_1000;
+        public int maxJoinItems = 50; // used to limit number of joined entries (from the join view) to decrease the response size
```

Comment:
Could you document in README too? 
We have a sample config there with comments/descriptions of some options.

Reply:
I was actually wondering why this needs to be configurable, just add complexity, while it is not likely to ever change.

Reply:
I missed the part, it should be also in the materialized view generation instead of hardcoded number (as it is now). I think it makes sense to have it configurable as we create join materialized views with number of data limited to the number defined in the config. For less resources we would be able even to decrease the number of joins

Reply:
I will add it to readme

Reply:
Ok, I thought we had it... Not sure where to document it. 
No, values.yaml is for helm/kubernetes setup, but you didn't make it helm-configurable, so it is not a right place.

Reply:
Added the parameter to materialzed view building too.

@ewelinagr let's then leave it as internal parameter. Otherwise, it would make sense to describe all the others too. What do you think?